development.html

```diff
@@ -51,18 +51,30 @@
 <h3>(Optional) As a Gradle project</h3>
 
 <p>
-    If you selected the Gradle option when creating the project, you can launch the Java server with Gradle:
+    If you selected the Gradle option when creating the project, you can launch the Java server with Gradle if you have installed it:
```

Comment:
I would write it differently, as I would recommend using the wrapper instead of installing Gradle